src/modules/WorkersPage/WorkersPage.tsx

```diff
@@ -0,0 +1,145 @@
+import { useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { useGetWorkers } from './hooks';
+import { StateType } from 'src/store';
+import { useTranslation } from 'react-i18next';
+import { Header, DetailsTile, Paragraph } from '@UG/libs/components';
+import { Worker } from '@UG/libs/types';
+import { styled, useTheme } from '@mui/material/styles';
+import { Box, TextField, Button } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { Link } from 'react-router-dom';
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledSearchTextField = styled(TextField)`
+  width: 710px;
+  & input {
+    font-family: 'Montserrat';
+    color: ${({ theme }) => theme.palette.primary.dark};
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+  & fieldset {
+    border-radius: 25px;
+  }
+  & .MuiOutlinedInput-root:hover {
+    & fieldset {
+      border: 1px solid ${({ theme }) => theme.palette.primary.main};
+    }
+  }
+  & .MuiFormLabel-root {
+    font-weight: 700;
+    // color: ${({ theme }) => theme.palette.secondary.main};
+  }
+  & label {
+    font-weight: 700;
+    margin-left: 30px;
+    color: ${({ theme }) => theme.palette.secondary.main};
+  }
+  & legend {
+    color: ${({ theme }) => theme.palette.primary.main};
+    margin-left: 30px;
+  }
+`;
+
+const StyledSearchButton = styled(Button)`
+  width: 60px;
+  height: 60px;
+  color: ${({ theme }) => theme.palette.primary.main};
+  border-radius: 25px;
+  font-size: 12px;
+`;
+
+interface StateProps {
+  isLoading: boolean;
+  workersList: Worker[];
+  errorMessage: string | null;
+}
+
+export const WorkersPage = () => {
```

Comment:
I think that this component could be renamed to sth more explicit like WorkersListPage